brave/brave-core
# WebTorrent: Add "Save All Files..." feature

Currently there is no ability to save all files at once. The user has to save each file separately. This PR fixes that.

Fixes: https://github.com/brave/brave-browser/issues/1230

## Submitter Checklist:

- Verified that these changes build without errors on
- Verified that these changes pass automated tests (unit, browser, security tests) on

## Test Plan:

1. Visit https://webtorrent.io/free-torrents
1. Click on the WIRED CD torrent
1. Click start download
1. Try to click on the disabled "Save All Files..." link *before* the download has completed. Confirm that it doesn't work.
1. Once the download is complete, click on "Save All Files...". It should prompt for a location to save the .zip file to.
1. Unzip the zip file however you do that on your OS and confirm that the torrent's files are inside.

## Reviewer Checklist:


## After-merge Checklist:

  changes has landed on.
